diff -Naurp lightdm-webkit-greeter-2.0.0.orig/src/lightdm-webkit-greeter.c lightdm-webkit-greeter-2.0.0/src/lightdm-webkit-greeter.c
--- lightdm-webkit-greeter-2.0.0.orig/src/lightdm-webkit-greeter.c	2016-01-10 20:42:31.000000000 +0100
+++ lightdm-webkit-greeter-2.0.0/src/lightdm-webkit-greeter.c	2017-01-29 11:04:52.468821070 +0100
@@ -532,13 +532,14 @@ authenticate_cb (JSContextRef context,
                          JSObjectRef thisObject, size_t argumentCount, const JSValueRef arguments[], JSValueRef * exception)
 {
   gchar *name = NULL;
+  GError *error = NULL;
 
   if (argumentCount > 0)
       name = arg_to_string (context, arguments[0], exception);
   else
       name = NULL;
 
-  lightdm_greeter_authenticate (GREETER, name);
+  lightdm_greeter_authenticate (GREETER, name, error);
 
   g_free (name);
   return JSValueMakeNull (context);
@@ -549,7 +550,8 @@ authenticate_as_guest_cb (JSContextRef c
                          JSObjectRef function,
                          JSObjectRef thisObject, size_t argumentCount, const JSValueRef arguments[], JSValueRef * exception)
 {
-  lightdm_greeter_authenticate_as_guest (GREETER);
+  GError *error = NULL;
+  lightdm_greeter_authenticate_as_guest (GREETER, error);
   return JSValueMakeNull (context);
 }
 
@@ -580,6 +582,7 @@ respond_cb (JSContextRef context,
             JSObjectRef thisObject, size_t argumentCount, const JSValueRef arguments[], JSValueRef * exception)
 {
   gchar *response;
+  GError *error = NULL;
 
   if (argumentCount != 1)
       return mkexception (context, exception, "Response not supplied");
@@ -588,7 +591,7 @@ respond_cb (JSContextRef context,
   if (!response)
     return JSValueMakeNull (context);
 
-  lightdm_greeter_respond (GREETER, response);
+  lightdm_greeter_respond (GREETER, response, error);
 
   g_free (response);
   return JSValueMakeNull (context);
@@ -599,7 +602,8 @@ cancel_authentication_cb (JSContextRef c
                           JSObjectRef function,
                           JSObjectRef thisObject, size_t argumentCount, const JSValueRef arguments[], JSValueRef * exception)
 {
-  lightdm_greeter_cancel_authentication (GREETER);
+  GError *error = NULL;
+  lightdm_greeter_cancel_authentication (GREETER, error);
   return JSValueMakeNull (context);
 }
 
@@ -723,15 +727,14 @@ set_language_cb (JSContextRef context,
                  JSObjectRef thisObject, size_t argumentCount, const JSValueRef arguments[], JSValueRef * exception)
 {
   gchar *language;
-
+  GError **error = NULL;
   if (argumentCount != 1)
       return mkexception (context, exception, "Language not supplied");
 
   language = arg_to_string (context, arguments[0], exception);
   if (!language)
     return JSValueMakeNull (context);
-
-  lightdm_greeter_set_language (GREETER, language);
+  lightdm_greeter_set_language (GREETER, language, error);
 
   g_free (language);
   return JSValueMakeNull (context);
