diff -Nuar Latte-Dock-116ea00ee0b0cf0d3bdf02ac68fa8b675a2138e4/CMakeLists.txt Latte-Dock-116ea00ee0b0cf0d3bdf02ac68fa8b675a2138e4-fixed/CMakeLists.txt
--- Latte-Dock-116ea00ee0b0cf0d3bdf02ac68fa8b675a2138e4/CMakeLists.txt	2017-03-09 15:21:24.000000000 +0000
+++ Latte-Dock-116ea00ee0b0cf0d3bdf02ac68fa8b675a2138e4-fixed/CMakeLists.txt	2017-03-11 06:02:33.509483194 +0000
@@ -57,6 +57,6 @@
 add_subdirectory(plasmoid)
 add_subdirectory(icons)
 
-plasma_install_package(build/shell/release org.kde.latte.shell shells shell)
-plasma_install_package(build/containment/release org.kde.latte.containment)
-plasma_install_package(build/plasmoid/release org.kde.latte.plasmoid)
+plasma_install_package(${CMAKE_BINARY_DIR}/shell/release org.kde.latte.shell shells shell)
+plasma_install_package(${CMAKE_BINARY_DIR}/containment/release org.kde.latte.containment)
+plasma_install_package(${CMAKE_BINARY_DIR}/plasmoid/release org.kde.latte.plasmoid)
