diff -Nuar Latte-Dock-116ea00ee0b0cf0d3bdf02ac68fa8b675a2138e4/containment/CMakeLists.txt Latte-Dock-116ea00ee0b0cf0d3bdf02ac68fa8b675a2138e4-fixed/containment/CMakeLists.txt
--- Latte-Dock-116ea00ee0b0cf0d3bdf02ac68fa8b675a2138e4/containment/CMakeLists.txt	2017-03-09 15:21:24.000000000 +0000
+++ Latte-Dock-116ea00ee0b0cf0d3bdf02ac68fa8b675a2138e4-fixed/containment/CMakeLists.txt	2017-03-11 12:33:24.744844394 +0000
@@ -1,2 +1,3 @@
+file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/release)
 file(COPY "contents" DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/release)
 configure_file(metadata.desktop.cmake release/metadata.desktop)
diff -Nuar Latte-Dock-116ea00ee0b0cf0d3bdf02ac68fa8b675a2138e4/plasmoid/CMakeLists.txt Latte-Dock-116ea00ee0b0cf0d3bdf02ac68fa8b675a2138e4-fixed/plasmoid/CMakeLists.txt
--- Latte-Dock-116ea00ee0b0cf0d3bdf02ac68fa8b675a2138e4/plasmoid/CMakeLists.txt	2017-03-09 15:21:24.000000000 +0000
+++ Latte-Dock-116ea00ee0b0cf0d3bdf02ac68fa8b675a2138e4-fixed/plasmoid/CMakeLists.txt	2017-03-11 12:34:01.714115463 +0000
@@ -1,2 +1,3 @@
+file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/release)
 file(COPY "contents" DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/release)
 configure_file(metadata.desktop.cmake release/metadata.desktop)
diff -Nuar Latte-Dock-116ea00ee0b0cf0d3bdf02ac68fa8b675a2138e4/shell/CMakeLists.txt Latte-Dock-116ea00ee0b0cf0d3bdf02ac68fa8b675a2138e4-fixed/shell/CMakeLists.txt
--- Latte-Dock-116ea00ee0b0cf0d3bdf02ac68fa8b675a2138e4/shell/CMakeLists.txt	2017-03-09 15:21:24.000000000 +0000
+++ Latte-Dock-116ea00ee0b0cf0d3bdf02ac68fa8b675a2138e4-fixed/shell/CMakeLists.txt	2017-03-11 12:34:34.693249346 +0000
@@ -1,3 +1,4 @@
+file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/release)
 file(COPY "contents" DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/release)
 configure_file(metadata.desktop.cmake release/metadata.desktop)
 
